src/d3a/models/strategy/__init__.py

```diff
@@ -430,12 +430,13 @@ def event_bid_deleted(self, *, market_id, bid):
             return
         self.remove_bid_from_pending(bid.id, market_id)
 
-    def event_bid_changed(self, *, market_id, existing_bid, new_bid):
+    def event_bid_split(self, *, market_id, original_bid, accepted_bid, residual_bid):
         assert ConstSettings.IAASettings.MARKET_TYPE is not 1, \
             "Invalid state, cannot receive a bid if single sided market is globally configured."
-        if new_bid.buyer != self.owner.name:
+        if accepted_bid.buyer != self.owner.name:
             return
-        self.add_bid_to_posted(market_id, bid=new_bid)
+        self.add_bid_to_posted(market_id, bid=accepted_bid)
```

Comment:
Clarification comment: the accepted bid needs to be added to posted, in order to be accepted in the trade event afterwards, right ? Thanks in advance

Reply:
This is probably not needed, but you skared me when you said: "What happens, if the trade wasnt successfull" So i added the accepted bid to the posted bids, which gets deleted once traded. What do you say? Shall i remove this feature?

Reply:
Doesn't hurt to have here IMO. By recalling the code flow in the trade event, this offer will move from posted to traded dict, therefore it has to be there. Thanks for clarifying